### PR TITLE
bcast-relay: T5313: verify() relay interfaces have IPv4 address configured

### PR DESCRIPTION
**Pull request by c-po**
## Change Summary

(cherry picked from commit ca7c063666c038d104082542f04ead6062e79246)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T5313

## Component(s) name

UDP broadcast relay

## How to test

Smoketests or

```
set service broadcast-relay id 1 description 'SONOS'
set service broadcast-relay id 1 interface 'eth3'
set service broadcast-relay id 1 interface 'eth4'
set service broadcast-relay id 1 interface 'eth5'
set service broadcast-relay id 1 port '1900'
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Review by jestabro — APPROVED** [review]

File: src/conf_mode/bcast_relay.py [hunk 1]
```diff
@@ -52,16 +54,14 @@ def verify(relay):
         if 'port' not in config:
             raise ConfigError(f'Port number mandatory for udp broadcast relay "{instance}"')
```
Comment (dmbaturin) [inline src/conf_mode/bcast_relay.py]: I think we should not omit prepositions, and protocol names should also be capitalzied: `Port number is mandatory for UDP broadcast relay`.

File: src/conf_mode/bcast_relay.py [hunk 2]
```diff
@@ -52,16 +54,14 @@ def verify(relay):
...
 
-        # if only oone interface is given it's a string -> move to list
-        if isinstance(config.get('interface', []), str):
-            config['interface'] = [ config['interface'] ]
         # Relaying data without two interface is kinda senseless ...
         if len(config.get('interface', [])) < 2:
             raise ConfigError('At least two interfaces are required for udp broadcast relay "{instance}"')
```
Comment (dmbaturin) [inline src/conf_mode/bcast_relay.py]: Could you capitalize UDP here as well?

**Review by dmbaturin — APPROVED** [review]
The code is fine of course, I only think that messages can be improved.

File: src/conf_mode/bcast_relay.py (hunk 2 above)
Comment (c-po) [reply src/conf_mode/bcast_relay.py]: Done

File: src/conf_mode/bcast_relay.py (hunk 1 above)
Comment (c-po) [reply src/conf_mode/bcast_relay.py]: Done

**Pull request merged**